### PR TITLE
Explicitly import React

### PR DESCRIPTION
**Pull request by sunnylqm**
Following this commit:
https://github.com/facebook/react-native/commit/a80dd9a92a8d7c201972d1a8b2392e9f893147c8

### CODE REVIEW

**Comment by vjeux** [comment]
cc @sema

**Comment by ide** [comment]
The 'react' import should be separate: import React from 'react'.

**Comment by sunnylqm** [comment]
Is `import React from 'react'` necessary for react native project? @ide

**Comment by dsibiski** [comment]
@sunnylqm Yes, check out this post as for the reasons why: https://www.facebook.com/groups/reactnativeoss/permalink/1540818949548067/

**Comment by sunnylqm** [comment]
Updated. Am I right then?

File: docs/Tutorial.md [hunk 1]
```diff
@@ -66,8 +66,10 @@ We're going to render the title, year, and thumbnail for the movie. Since thumbn
 
 ```javascript
 import React, {
+  Component
```
Comment (ide) [inline docs/Tutorial.md]: always add trailing commas

**Comment by mkonicek** [comment]
c

**Comment by mkonicek** [comment]
c

**Comment by ide** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**